### PR TITLE
Update pom.xml with latest version of flyway

### PR DESCRIPTION
**Pull request by vishalkumar-barnwal**
https://internal.almoctane.com/ui/?p=131002/6001#/entity-navigation?entityType=work_item&id=517060

### CODE REVIEW

**Review by michael-bryson — APPROVED** [review]

**Comment by michael-bryson** [comment]
Update the release notes.

File: release-notes-1.1.0.md [hunk 1]
```diff
@@ -4,5 +4,10 @@
 ${version-number}
 
 #### New Features
+- None
+
+#### Patch Fixes Included
+- US517060 - Upgrade Flyway version
```
Comment (michael-bryson) [inline release-notes-1.1.0.md]:
```suggestion
- US517060 - Flyway version upgraded to [9.3.1](https://flywaydb.org/documentation/learnmore/releaseNotes#9.3.1)
```

File: release-notes-1.1.0.md [hunk 2]
```diff
@@ -4,5 +4,10 @@
 ${version-number}
 
 #### New Features
+- None
+
+#### Patch Fixes Included
+- US517060 - Flyway version upgraded to [9.3.1](https://flywaydb.org/documentation/learnmore/releaseNotes#9.3.1)
```
Comment (dermot-hardy) [inline release-notes-1.1.0.md:7]: This upgrade feels more like a new feature than a patch fix to me.

Comment (vishalkumar-barnwal) [reply release-notes-1.1.0.md:7]: I have updated the release note and put this update under new feature.

**Review by dermot-hardy — APPROVED** [review]

**Pull request merged**